openshift/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.centos.org/centos/centos:7
+
+LABEL maintainer="thrcka@redhat.com"
+
+ENV LANG=en_US.UTF-8
+
+RUN yum --setopt=tsflags=nodocs install -y epel-release centos-release-openshift-origin && \
+    yum --setopt=tsflags=nodocs install -y https://download.postgresql.org/pub/repos/yum/9.4/redhat/rhel-7-x86_64/pgdg-centos94-9.4-3.noarch.rpm &&\
```

Comment:
Not sure I understand what it is and why it can't be installed from centos/epel repo.
Could you add a comment there (in the Dockerfile) ?